src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { BsGithub,BsTwitter,BsLinkedin } from "react-icons/bs";
+import "../Footer/Footer.css";
+
+ export const Footer = () => {
+  return (
+   <div className="footer">
+      <div className="footer-heading">Made by Mariya Sada | © | 2022</div>
+      <ul className="nav-items">
+        <li className="nav-item">
+          <a href="https://github.com/mariyasada">
+            <span className="nav-icon">              
+             <BsGithub title="Github" />
+            </span>
+          </a>
```

Comment:
You have imported the link component but have not used it.

Reply:
yes when i use Link instead of a it doesn't redirect to social website that's why i a\have not used